android_openssl.cmake

```diff
@@ -0,0 +1,20 @@
+function(add_android_openssl_libraries_to)
```

Comment:
having the "_to" at the end of the name feels a bit awkward, I think it better be removed